### PR TITLE
Adding 1.23 support

### PR DESCRIPTION
**Pull request by boyamurthy**
This PR contains

1. New sample file for k8 1.23
2. Updated K8 supported versions 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|    #[136  ](https://github.com/dell/csm/issues/136)        |

# Checklist:

- [X] Have you run format,vet & lint checks against your submission?
- [X] Have you made sure that the code compiles?
- [ ] Did you run the unit & integration tests successfully?
- [ ] Have you maintained at least 90% code coverage?
- [ ] Have you commented your code, particularly in hard-to-understand areas
- [ ] Have you done corresponding changes to the documentation
- [ ] Did you run tests in a real Kubernetes cluster?
- [ ] Backward compatibility is not broken

### CODE REVIEW

File: helm/csi-powermax/k8s-1.23-values.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # IT IS RECOMMENDED YOU DO NOT CHANGE THE IMAGES TO BE DOWNLOADED.
-kubeversion: "v1.20"
+kubeversion: "v1.23"
```
Comment (prablr79) [inline helm/csi-powermax/k8s-1.23-values.yaml:3]: it would be good to assess the sidecars and arrive at a common ground.

**Review by prablr79 — CHANGES_REQUESTED** [review]

File: helm/csi-powermax/k8s-1.23-values.yaml (hunk 1 above)
Comment (boyamurthy) [reply helm/csi-powermax/k8s-1.23-values.yaml:3]: hi @prablr79  , I checked that no new versions of sidecars available . Right now we are using all the latest .

Comment (prablr79) [reply helm/csi-powermax/k8s-1.23-values.yaml:3]: @boyamurthy  please do check there is new sidecar need to update https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v2.4.0

Comment (boyamurthy) [reply helm/csi-powermax/k8s-1.23-values.yaml:3]: Thanks @prablr79  , updated the sidecar .

Comment (prablr79) [reply helm/csi-powermax/k8s-1.23-values.yaml:3]: @boyamurthy looks like the changes still not reflected.

**Review by prablr79 — APPROVED** [review]

**Comment by prablr79** [comment]
@boyamurthy  please make sure you test the same version side cars for previous K8 versions.

**Comment by boyamurthy** [comment]
> @boyamurthy please make sure you test the same version side cars for previous K8 versions.

yes @prablr79  , will use the same side car version for k8 1.21 and 1.22 .

**Pull request merged**